From 515617e55be9a7bfa738a9c32ef8b19065de37d4 Mon Sep 17 00:00:00 2001
From: aavit <eirik.aavitsland@digia.com>
Date: Fri, 22 Nov 2013 15:49:44 +0100
Subject: [PATCH] Recognize newer libmng versions in config test

libmng 2.0.x has been released and is compatible and usable, but since
it no longer provides a VERSION_MAJOR macro, the config test would fail.

Task-number: QTBUG-34894
Change-Id: I36f6ed9d69dbae88feb1b88ce099bf36c9283133
Reviewed-by: Liang Qi <liang.qi@digia.com>
(cherry picked from qtimageformats/9ae386653c321c8ddc10fad5ea88f32ebb3d3ffe)
---
 config.tests/unix/libmng/libmng.cpp |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/config.tests/unix/libmng/libmng.cpp b/config.tests/unix/libmng/libmng.cpp
index 0fbe554..9db10ff 100644
--- a/config.tests/unix/libmng/libmng.cpp
+++ b/config.tests/unix/libmng/libmng.cpp
@@ -46,9 +46,11 @@ int main(int, char **)
     mng_handle hMNG;
     mng_cleanup(&hMNG);
 
+#if defined(MNG_VERSION_MAJOR)
 #if MNG_VERSION_MAJOR < 1 || (MNG_VERSION_MAJOR == 1 && MNG_VERSION_MINOR == 0 && MNG_VERSION_RELEASE < 9)
 #error System libmng version is less than 1.0.9; using built-in version instead.
 #endif
+#endif
 
     return 0;
 }
-- 
1.7.1

